release-notes/release-notes58.md

```diff
@@ -27,6 +27,12 @@ The following new features were added for the {{site.productname}} 5.8 release.
 
 ### Feature name
 
+### Added new `fixed_toolbar_container_target` setting
+
+Added a new `fixed_toolbar_container_target` setting which renders the toolbar in the specified `HTMLElement`. Patch contributed by [pvrobays](https://github.com/pvrobays).
```

Comment:
How about something like

```suggestion
The new `fixed_toolbar_container_target` option allows the toolbar to be rendered in a specified `HTMLElement`. This feature was contributed by [pvrobays](https://github.com/pvrobays).
```

Reply:
Yeah, this looks good to me and matches what's normally used for release notes. We may also want to mention it's similar to the `fixed_toolbar_container` option here just so people are aware when reading this without going to have to read the full details in the configuration docs.